sdk/identity/azure-identity/azure/identity/_credentials/azure_cli.py

```diff
@@ -90,21 +93,21 @@ def get_token(self, *scopes: str, **kwargs: Any) -> AccessToken:
         resource = _scopes_to_resource(*scopes)
         command = COMMAND_LINE.format(resource)
         tenant = resolve_tenant(
-            default_tenant=self.tenant_id,
-            additionally_allowed_tenants=self._additionally_allowed_tenants,
-            **kwargs
+            default_tenant=self.tenant_id, additionally_allowed_tenants=self._additionally_allowed_tenants, **kwargs
         )
         if tenant:
             command += " --tenant " + tenant
-        output = _run_command(command, self._process_timeout)
+        output = _run_command(command, self._process_timeout, is_chained=self._is_chained)
 
         token = parse_token(output)
         if not token:
             sanitized_output = sanitize_output(output)
-            raise ClientAuthenticationError(
-                message="Unexpected output from Azure CLI: '{}'. \n"
-                        "To mitigate this issue, please refer to the troubleshooting guidelines here at "
-                        "https://aka.ms/azsdk/python/identity/azclicredential/troubleshoot.".format(sanitized_output))
+            message = "Unexpected output from Azure CLI: '{}'. \n"
```

Comment:
woops, looks like format is on the wrong line. Can maybe just f string the first line here?  `{sanitized_output}`